### PR TITLE
Arpit details

### PR DESCRIPTION
**Pull request by ausome197**

### CODE REVIEW

File: assets/data/team.json [hunk 1]
```diff
@@ -1,4 +1,38 @@
 [
+<<<<<<< HEAD
```
Comment (YashKumarVerma) [inline assets/data/team.json:2]: unwanted line

Comment (ausome197) [reply assets/data/team.json]: yes it was due to merge conflicts I have resolved it

File: assets/data/team.json [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+        "email":"vivekshah9969@gmail.com",
+        "github_link": "https://github.com/vivekshah1801",
+        "linkedIn_link": "https://www.linkedin.com/in/vivekshah1801/"
+    },
+    {
+        "name":"vipul krishna",
+        "role":"contributor",
+        "profile_pic":"../assets/team/vipul.jpg",
+        "email":"vipul07.mathuria@gmail.com",
+        "github_link":"https://github.com/vipul1999",
```
Comment (YashKumarVerma) [inline assets/data/team.json:24]: not part of source. please update your fork.

Comment (YashKumarVerma) [reply assets/data/team.json:24]: these lines have been updated.

Comment (ausome197) [reply assets/data/team.json]: done

**Pull request merged**